.ci/deps.sh

```diff
@@ -67,8 +67,8 @@ fi
 
 deps_perl="perl libperl-critic-perl"
 
-sudo apt-get -y update
-sudo apt-get -y --no-install-recommends install $deps $deps_perl $deps_infer
+sudo apt-get -y update|grep -v '\%'
```

Comment:
Moreover redirecting the data to /dev/null will completely silent the data. We don't want that. Output with `grep -v '%'` will be like this. 
![image](https://cloud.githubusercontent.com/assets/17270966/22258693/9aa8558c-e288-11e6-8c7e-30ee9c5defb2.png)
But with redirecting it to /dev/null the output will be null